### PR TITLE
Add basic vulnerability summary documentation

### PR DESCRIPTION
**Pull request by spiffcs**
@wagoodman had a very good comment explaining our vulnerability summary and I didn't want it to get lost on the issue so made a draft for our docs folder. 

If this is not the correct spot we can discuss where a better place to add this information might be.

Signed-off-by: Christopher Angelo Phillips <christopher.phillips@anchore.com>

### CODE REVIEW

**Review by kzantow — APPROVED** [review]
Might expand the basic shape to be the entire Grype JSON output tree, but great info as is!

File: docs/vulnerability.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+## Vulnerability Summary
+
+### Basic Grype Vulnerability Data Shape
+````json
```
Comment (wagoodman) [inline docs/vulnerability.md:4]: one too many `

File: docs/vulnerability.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+#### Language
+This section tries to explain the language ecosystem where the vulnerability was found ex: `CVE-2015-5237` is a vulnerability related to the Golang protobuf library so the language in this case would be "go".
+
+#### Licenses
+This section tries to enumerate all discovered licenses found associated with the package described under the `Artifact` section.
+
+#### CPES
+This section lists all CPE used to matched against vulnerability data found in the grype vulnerability database.
+
+#### PURL
```
Comment (wagoodman) [inline docs/vulnerability.md:39]: should we delete purl and metadata sections?

File: docs/vulnerability.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+Information pertaining to vulnerabilities found to be related to the main reported vulnerability. Maybe the vulnerability we matched on was a GitHub Security Advisory, which has an upstream CVE (in the authoritative national vulnerability database). In these cases we list the upstream vulnerabilities here.
+
+#### MatchDetails
+This section tries to explain what we searched for while looking for a match and exactly what details on the package and vulnerability that lead to a match.
+
+#### Artifact 
+This is a subset of the information that we know about the package (when compared to the [Syft](https://github.com/anchore/syft****) json output, we summarize the metadata section). 
+This has information about where within the container image or directory we found the package, what kind of package it is, licensing info, pURLs, CPEs, etc.
+
+#### Language
```
Comment (wagoodman) [inline docs/vulnerability.md:30]: language, licenses, and CPEs should be `#####` (under "Artifacts") not `####`

File: docs/vulnerability.md (hunk 2 above)
Comment (spiffcs) [reply docs/vulnerability.md:39]: I can remove these sections for now.

File: docs/vulnerability.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+All information on the specific vulnerability that was directly matched on (e.g. ID, severity, CVSS score, fix information, links for more information).
+
+#### RelatedVulnerabilities
+Information pertaining to vulnerabilities found to be related to the main reported vulnerability. Maybe the vulnerability we matched on was a GitHub Security Advisory, which has an upstream CVE (in the authoritative national vulnerability database). In these cases we list the upstream vulnerabilities here.
+
+#### MatchDetails
+This section tries to explain what we searched for while looking for a match and exactly what details on the package and vulnerability that lead to a match.
+
+#### Artifact 
+This is a subset of the information that we know about the package (when compared to the [Syft](https://github.com/anchore/syft****) json output, we summarize the metadata section).
```
Comment (wagoodman) [inline docs/vulnerability.md:27]:
```suggestion
This is a subset of the information that we know about the package (when compared to the [Syft](https://github.com/anchore/syft) json output, we summarize the metadata section). 
```

File: docs/vulnerability.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+This section tries to enumerate all discovered licenses found associated with the package described under the `Artifact` section.
+
+#### CPES
+This section lists all CPE used to matched against vulnerability data found in the grype vulnerability database.
+
+#### PURL
+
+#### Metadata
+
+#### Interesting subsections to investigate
```
Comment (wagoodman) [inline docs/vulnerability.md:43]: does this section need to be in this doc? this seems more related to the issue that this was on than this doc

**Comment by wagoodman** [comment]
Should we make this multi-section document into a single-section bulleted list and put in the README.md instead? Each section is pretty small and it would make the information more discoverable in the README.

**Comment by spiffcs** [comment]
> Should we make this multi-section document into a single-section bulleted list and put in the README.md instead? Each section is pretty small and it would make the information more discoverable in the README.

Sure! I can reformat it that way with the other edits.

**Comment by spiffcs** [comment]
@wagoodman updated and folded into `README.md`

**Review by wagoodman — APPROVED** [review]

**Pull request merged**